### DISCUSSION TITLE
Appropriate data points for bar graph

### DISCUSSION THREAD

**Discussion by bfamzz** [lcallarec/live-chart#30] [Q&A]
@lcallarec  please can you help with the correct data points for bar graph?

The end goal is to have something like this.

<img width="138" alt="live-chart-bar-graph (2)" src="https://user-images.githubusercontent.com/38426051/153955852-6e6bb542-85df-4790-a2d1-860fe7aa4851.png">

with the name of of each data point showing below each bar and with the corresponding height.

Thanks.

**Comment by lcallarec** [comment]
Hi @BFamzz ! Thanks for asking. Unfortunately, live-chart's primary goal is to display time-based series, but there was some works done to handle static charts. Stay tuned !
